plugins/faustwp/includes/replacement/graphql-callbacks.php

```diff
@@ -14,7 +14,6 @@
 	exit;
 }
 
-add_filter( 'graphql_request_results', __NAMESPACE__ . '\\url_replacement' );
```

Comment:
Also still testing this removal to ensure nothing broke. The tests pass, but we need to confirm we're not breaking things we're not testing.